azavea/temperate
# Define verbose labels for PlanIt models

## Overview
Because Django can use model names in auto-generated error messages, we want to define human-friendly names for certain models to ensure the messages look professional.

### Demo
<img width="359" alt="screen shot 2018-02-27 at 10 54 40 am" src="https://user-images.githubusercontent.com/1032849/36738950-c13f5e3a-1bac-11e8-9eee-0550c3eab201.png">
<img width="623" alt="screen shot 2018-02-27 at 11 08 07 am" src="https://user-images.githubusercontent.com/1032849/36739686-8b4d44fc-1bae-11e8-864b-c7b7dc5afa6d.png">
<img width="615" alt="screen shot 2018-02-27 at 11 07 43 am" src="https://user-images.githubusercontent.com/1032849/36739687-8b59aaa8-1bae-11e8-81bc-4e06bb818c96.png">
<img width="637" alt="screen shot 2018-02-27 at 10 53 51 am" src="https://user-images.githubusercontent.com/1032849/36738952-c150a726-1bac-11e8-89d6-de68fd398a4b.png">

## Testing Instructions
- Open the admin panel
  - You should not see sections labeled by "Plan It"

Closes #652
